vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentInformationDialog.java

```diff
@@ -0,0 +1,166 @@
+  private final List<ComponentEventListener<CancelEvent<ExperimentInformationDialog>>> cancelEventListeners = new ArrayList<>();
+  private final List<ComponentEventListener<ConfirmEvent<ExperimentInformationDialog>>> confirmEventListeners = new ArrayList<>();
+  private final MODE mode;
+
+  public ExperimentInformationDialog(
+      ExperimentalDesignSearchService experimentalDesignSearchService) {
+    this(experimentalDesignSearchService, false);
+  }
+
+  private ExperimentInformationDialog(
+      ExperimentalDesignSearchService experimentalDesignSearchService, boolean mode) {
+    super();
+    this.mode = mode ? MODE.EDIT : MODE.ADD;
+    addClassName("experiment-information-dialog");
+    defineExperimentComponent = new DefineExperimentComponent(experimentalDesignSearchService);
+    layoutComponent();
+    initDialogueContent();
+    configureComponent();
+  }
+
+  private void layoutComponent() {
+    setHeaderTitle(TITLE);
+    add(defineExperimentComponent);
+    setConfirmButtonLabel("Add");
+    setCancelButtonLabel("Cancel");
+    final DialogFooter footer = getFooter();
+    footer.add(this.cancelButton, this.confirmButton);
+  }
+
+  private void initDialogueContent() {
+    add(defineExperimentComponent);
+  }
+
+  private void configureComponent() {
+    configureCancelling();
+    configureConfirmation();
+  }
+
+  /**
+   * Creates a new dialog prefilled with experiment information.
+   *
+   * @param experimentalDesignSearchService the service providing the selectable options for the
+   *                                        analyte, specimen and species within this dialog
+   * @param experimentName                  experimentName to be preset within the dialog
+   * @param species                         List of {@link Species} to be preset within the dialog
+   * @param specimen                        List of {@link Specimen} to be preset within the dialog
+   * @param analytes                        List of {@link Analyte} to be preset within the dialog
+   * @return a new instance of the dialog
+   */
+  public static ExperimentInformationDialog prefilled(
+      ExperimentalDesignSearchService experimentalDesignSearchService,
+      String experimentName, Collection<Species> species, Collection<Specimen> specimen,
+      Collection<Analyte> analytes) {
+    return editDialog(experimentalDesignSearchService, experimentName, species, specimen, analytes);
+  }
+
+  private static ExperimentInformationDialog editDialog(
+      ExperimentalDesignSearchService experimentalDesignSearchService, String experimentName,
+      Collection<Species> species, Collection<Specimen> specimen, Collection<Analyte> analytes) {
+    ExperimentInformationDialog experimentInformationDialog = new ExperimentInformationDialog(
+        experimentalDesignSearchService, true);
+    experimentInformationDialog.setExperimentInformation(experimentName, species, specimen,
+        analytes);
+    return experimentInformationDialog;
+  }
```

Comment:
what is the difference between prefilled and editDialog?

Reply:
`prefilled()` is the public accessor method to be used if one wants to create an ExperimentalVariablesDialog in edit mode. 
I tried to adhere to the structure introduced in the `ExperimentalVariablesDialog` (see screenshot below) so the dialogues can be created in  the same way. 🤔 
![Bildschirmfoto 2023-08-08 um 10 20 45](https://github.com/qbicsoftware/data-manager-app/assets/29627977/2924960a-6051-4a26-8140-0f8c29e6fe15)